### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by SumitSaundadkar**

### CODE REVIEW

**Comment by rahulyadav139** [comment]
1. Add some padding to the number badge
2. Green color badge has some placement issues. Kindly look into it.

**Comment by saurabhchirde** [comment]
You could have used the same width for all alerts.

**Comment by ShrutiLakhani** [comment]
1. Since we are already adding width: max-content; to alert class, I think there is no need to add it to individual variants.
2. I personally feel, adding a dismiss icon to the right and may be a variant specific icon (like a tick for success) on the left (before the text) would make it more presentable. 
3. As Saurabh mentioned, better to have consistent width.

**Comment by SumitSaundadkar** [comment]
> 1. Since we are already adding width: max-content; to alert class, I think there is no need to add it to individual variants.
> 2. I personally feel, adding a dismiss icon to the right and may be a variant specific icon (like a tick for success) on the left (before the text) would make it more presentable.
> 3. As Saurabh mentioned, better to have consistent width.

yes updated sarabh and shruti thanks

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+
+
+.alerttypes{
+    width: 80%;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    
+    background-color:var(--offline-color);
+    padding: 2rem;
```
Comment (tarunsankhla) [inline components/alert.css]: Try using px for margin and padding coz general we don't want to enlarge them as screen enlarged

**Review by tarunsankhla — APPROVED** [review]
Looks good 
In preview the badge looks bigger then the icon bcoz we generally prefer badges to be smaller then the container it is attached too

**Review by m26unkwn — APPROVED** [review]
Everything is good to merge.

**Pull request merged**